README.md

```diff
@@ -850,7 +850,20 @@ Adding the `--with-netlify-static` flag to the build script will produce static
 }
 ```
 
-### Cloudflare (experimental)
+### Cloudflare Pages (experimental)
+
+**Note:** the current integration with Cloudflare Pages requires a `waku.config.ts` file to be
+added at the root of your project, with the basePath set like so:
+
+```
+export default {
+  basePath: "/public/"
```

Comment:
Yes I have no idea why this happens either. It could be from CFP, Waku, or Vite's build process. Or even a combination, e.g. CFP might not be able to read `import.meta.env` in some cases.

I definitely see it as temporary. My suggestion is, seeing as Waku doesn't currently have lots of users, we accept an initial hacky workaround. I will then use the published package to see if anything behaves differently and try working on a better solution.